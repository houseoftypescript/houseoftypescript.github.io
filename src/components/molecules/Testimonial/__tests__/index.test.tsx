import { render } from '@testing-library/react';
import { Testimonial } from '../index';

describe('Testimonial', () => {
  it('renders correctly', () => {
    const button = render(<Testimonial></Testimonial>);
    expect(button.container).toMatchSnapshot();
  });
});
