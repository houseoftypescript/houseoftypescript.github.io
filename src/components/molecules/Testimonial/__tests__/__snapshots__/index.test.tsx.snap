// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial renders correctly 1`] = `
<div>
  <div
    class="pb-16"
  >
    <div
      class="bg-no-repeat w-full md:w-2/3 pl-0 md:pl-16 mx-auto"
      style="background-image: url(/img.jpg);"
    >
      <blockquote
        class="pb-8"
      >
        <h2
          class="font-b612 text-2xl text-gray-700 text-justify leading-loose"
        />
      </blockquote>
      <p
        class="font-medium text-gray-500 text-lg"
      />
    </div>
  </div>
</div>
`;
